lib/stimulus_reflex/channel.rb

```diff
@@ -52,6 +53,13 @@ def receive(data)
 
   private
 
+  def compute_reflex_name(reflex_name)
+    raise unless is_reflex?(reflex_name.constantize)
+    reflex_name
+  rescue
+    "#{reflex_name}Reflex"
```

Comment:
You can use `const_defined?` for this:

```suggestion
    Object.const_defined?(reflex_name.to_s) ? reflex_name : "#{reflex_name}Reflex"
```

Needs to be scoped to `Object` to ensure the lookup path is what you'd expect. And _needs_ to be a String, hence the `to_s` (technically, anything that responds `to_str`.)

Reply:
There needs to be a #clearlydoesntknowtherightfunction tag. You are very kind. Obviously you are right!

Reply:
@foca we went in a different, simpler direction with this but I really appreciate learning about const_defined? !

Reply:
The resulting code is much simpler overall! I digs.